tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -626,6 +629,115 @@ void test_spacetime_derivative_of_spacetime_metric(
+    // +y direction
+    get<1>(tmp) = 0.5;
+    for (size_t i = 0; i < VolumeDim; ++i) {
+      for (size_t j = 0; j < VolumeDim; ++j) {
+        get<0>(tmp)[i * VolumeDim + j] =
+            lower_bound[0] + 0.5 * static_cast<double>(i);
+        get<2>(tmp)[i * VolumeDim + j] =
+            lower_bound[2] + 0.5 * static_cast<double>(j);
+      }
+    }
+    return tmp;
+  }();
+
+  auto local_inverse_spatial_metric =
+      make_with_value<tnsr::II<DataVector, VolumeDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+  auto local_phi =
+      make_with_value<tnsr::iaa<DataVector, VolumeDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+  auto local_d_pi =
+      make_with_value<tnsr::iaa<DataVector, VolumeDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+  auto local_d_phi =
+      make_with_value<tnsr::ijaa<DataVector, VolumeDim, Frame::Inertial>>(
+          inertial_coords, 0.);
+
+  // Setting inverse_spatial_metric
+  for (size_t i = 0; i < get<0>(inertial_coords).size(); ++i) {
+    for (size_t j = 0; j < VolumeDim; ++j) {
+      local_inverse_spatial_metric.get(0, j)[i] = 41.;
+      local_inverse_spatial_metric.get(1, j)[i] = 43.;
+      local_inverse_spatial_metric.get(2, j)[i] = 47.;
+    }
+  }
+  // Setting pi AND phi
+  for (size_t i = 0; i < get<0>(inertial_coords).size(); ++i) {
+    for (size_t a = 0; a <= VolumeDim; ++a) {
+      for (size_t b = 0; b <= VolumeDim; ++b) {
+        // local_pi.get(a, b)[i] = 1.;
+        local_phi.get(0, a, b)[i] = 3.;
+        local_phi.get(1, a, b)[i] = 5.;
+        local_phi.get(2, a, b)[i] = 7.;
+      }
+    }
+  }
+  // Setting local_d_phi
+  // initialize dPhi (with same values as for SpEC)
+  for (size_t i = 0; i < get<0>(inertial_coords).size(); ++i) {
+    for (size_t a = 0; a <= VolumeDim; ++a) {
+      for (size_t b = 0; b <= VolumeDim; ++b) {
+        local_d_pi.get(0, a, b)[i] = 1.;
+        local_d_phi.get(0, 0, a, b)[i] = 3.;
+        local_d_phi.get(0, 1, a, b)[i] = 5.;
+        local_d_phi.get(0, 2, a, b)[i] = 7.;
+        local_d_pi.get(1, a, b)[i] = 53.;
+        local_d_phi.get(1, 0, a, b)[i] = 59.;
+        local_d_phi.get(1, 1, a, b)[i] = 61.;
+        local_d_phi.get(1, 2, a, b)[i] = 67.;
+        local_d_pi.get(2, a, b)[i] = 71.;
+        local_d_phi.get(2, 0, a, b)[i] = 73.;
+        local_d_phi.get(2, 1, a, b)[i] = 79.;
+        local_d_phi.get(2, 2, a, b)[i] = 83.;
+      }
+    }
+  }
+
+  // Call tested function
+  auto local_ricci_3 = GeneralizedHarmonic::spatial_ricci_tensor(
+      local_phi, local_d_phi, local_inverse_spatial_metric);
+
+  // Initialize with values from SpEC
+  auto spec_ricci_3 = local_ricci_3;
+  get<0, 0>(spec_ricci_3) = 153230.;
+  get<0, 1>(spec_ricci_3) = 5283.;
+  get<0, 2>(spec_ricci_3) = -142541.;
+  get<1, 0>(spec_ricci_3) = 5283.;
+  get<1, 1>(spec_ricci_3) = 2497.;
+  get<1, 2>(spec_ricci_3) = -928.;
+  get<2, 2>(spec_ricci_3) = 141189.;
+
+  // Compare values returned by BC action vs those from SpEC
```

Comment:
what is BC?

Reply:
this should have read "Compare with values from SpEC". fixed it.